lua/wikis/rainbowsix/Infobox/Character/Custom.lua

```diff
@@ -0,0 +1,484 @@
+						HtmlWidgets.Small{
+							children = { args['eliteskin-date'] }
+						}
+					}
+				} or nil
+			)
+		},
+		Cell{
+			name = 'Availability',
+			content = { args.availability }
+		}
+	)
+end
+
+---@param input string
+---@return Widget[]|nil
+function CustomCharacter:_getPriceCells(input)
+	local lowerInput = input:lower()
+	if (lowerInput == 'launch') then
+		local lang = mw.getContentLanguage()
+		return {
+			Cell{
+				name = 'Renown price',
+				content = WidgetUtil.collect(
+					'Free',
+					Array.map({ 500, 1000, 1500, 2000 }, function (renownPrice)
+						return HtmlWidgets.Fragment{
+							children = {
+								lang:formatNum(renownPrice),
+								' ',
+								STANDARD_EDITION_LABEL
+							}
+						}
+					end),
+					HtmlWidgets.Fragment{
+						children = {
+							'10% off ',
+							SEASON_PASS_LABEL
+						}
+					}
+				),
+			},
+			self:_getPriceCell('Renown', OPERATOR_PRICES.launch.credit)
+		}
+	elseif Logic.isNotEmpty(OPERATOR_PRICES[lowerInput]) then
+		local priceData = OPERATOR_PRICES[lowerInput]
+		return {
+			self:_getPriceCell('Renown', priceData.renown),
+			self:_getPriceCell('R6 Credit', priceData.credit)
+		}
+	end
+	return nil
+end
+
+---@param currency string
+---@param price integer
+---@return Widget
+function CustomCharacter:_getPriceCell(currency, price)
+	local lang = mw.getContentLanguage()
+	return Cell{
+		name = currency .. ' price',
+		content = {
+			lang:formatNum(price),
+			HtmlWidgets.Fragment{
+				children = {
+					lang:formatNum(price * 0.9),
+					' ',
+					SEASON_PASS_LABEL
+				}
+			}
+		},
+	}
+end
+
+---@param args table<string, any>
+---@return Widget[]
+function CustomCharacter:_getBaseStats(args)
+	return WidgetUtil.collect(
+		Title{ children = 'Base Stats' },
+		Cell{
+			name = 'Armor/Health',
+			content = self:_getArmorContent((args.armor or ''):lower()),
+			options = { separator = ' ' }
+		},
+		Cell{
+			name = 'Speed',
+			content = self:_getSpeedContent((args.speed or ''):lower()),
+			options = { separator = ' ' }
+		},
+		Cell{
+			name = 'Difficulty',
+			content = self:_getDifficultyContent((args.difficulty or ''):lower()),
+			options = { separator = ' ' }
+		}
+	)
+end
+
+---@param armor string
+---@return (string|Widget)[]
+function CustomCharacter:_getArmorContent(armor)
+	local armorContent = FnUtil.curry(CustomCharacter._getStatContent, 'armor')
+	local armorEq = FnUtil.curry(Operator.eq, armor)
+	if Array.any({'1', 'light', 'low'}, armorEq) then
+		return armorContent(1, 'Light (100 HP)')
+	elseif Array.any({'2', 'medium'}, armorEq) then
+		return armorContent(2, 'Medium (110 HP)')
+	elseif Array.any({'3', 'slow', 'high'}, armorEq) then
+		return armorContent(3, 'Heavy (125 HP)')
+	end
+	return {}
+end
+
+---@param speed string
+---@return (string|Widget)[]
+function CustomCharacter:_getSpeedContent(speed)
+	local speedContent = FnUtil.curry(CustomCharacter._getStatContent, 'speed')
+	local speedEq = FnUtil.curry(Operator.eq, speed)
```

Comment:
Unless something changed in the last year, speed and armor (now just health actually) are inverse of eachother 3/1, 2/2, 1/3 are the only possible combos. We could look at  having a single input for it. Usually referred to by speed.